src/blockchain_utilities/blocksdat_file.h

```diff
@@ -31,8 +31,8 @@
 #include <boost/iostreams/stream_buffer.hpp>
 #include <boost/iostreams/stream.hpp>
 #include <boost/iostreams/device/back_inserter.hpp>
-
 #include <boost/iostreams/filtering_streambuf.hpp>
+#include <boost/filesystem.hpp>
```

Comment:
`#include <boost/filesystem/path.hpp>` might work here too ...

Reply:
Also needs operations.hpp, but that's still a win.